pkg/minikube/update/update.go

```diff
@@ -14,38 +14,46 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package notify
+package update
 
 import (
+	"crypto"
+	"encoding/hex"
 	"encoding/json"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"net/http"
+	"os"
+	"runtime"
 	"strings"
+	"syscall"
 	"time"
 
 	"github.com/blang/semver"
+	"github.com/bugsnag/osext"
 	"github.com/golang/glog"
+	update "github.com/inconshreveable/go-update"
 	"github.com/spf13/viper"
 	"k8s.io/minikube/pkg/minikube/config"
 	"k8s.io/minikube/pkg/minikube/constants"
 	"k8s.io/minikube/pkg/version"
 )
 
-const updateLinkPrefix = "https://github.com/kubernetes/minikube/releases/tag/v"
+const downloadLinkFormat = "https://github.com/kubernetes/minikube/releases/download/v%s/%s"
```

Comment:
I forgot about that. I always use API tokens so don't hit those limits but most users probably won't do that.

Yeah updating release json makes a lot of sense. Need to automate the release process to add this in but will document it for now.